pandas/core/algorithms.py

```diff
@@ -112,16 +112,19 @@
 # --------------- #
 def _ensure_data(values: ArrayLike) -> np.ndarray:
     """
-    routine to ensure that our data is of the correct
-    input dtype for lower-level routines
+    Ensure values is of the correct input dtype for lower-level routines.
 
     This will coerce:
     - ints -> int64
     - uint -> uint64
```

Comment:
i think the ints and uints are unchanged

Reply:
i didn't yet check those. will look tomorrow.